ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/OldApiUtils.java

```diff
@@ -112,6 +118,9 @@ static void handleOldApiResponse(final Server server, final RoutingContext routi
         response.end(responseBody);
       }
     }
+    pullQueryMetrics
```

Comment:
This is the only place where we have access to the response sent over the wire. That's why I had to pass the `pullQueryMetrics` object in here